test/index.js

```diff
@@ -240,14 +320,22 @@ describe('normalize.undefined', function() {
     expect(result).toEqual(value);
     done();
   });
+
+  it('rejects values that won\'t coerce to undefined', function(done) {
+    var value = 'invalid';
+    var result = normalize.boolean(value);
+    expect(result).toEqual(null);
+    done();
+  });
 });
 
 describe('normalize.date', function() {
 
   it('compares value to typeof number', function(done) {
```

Comment:
You mean the test descriptions with `compares to typeof`? I thought the same, but then I wasn't sure what to change it to that would keep clear the distinction between, e.g., `typeof number` and `instanceof Number`. I thought only the uppercase would be too minute. Do you have a suggestion?